docs/azure/services/Azure_Functions.md

```diff
@@ -0,0 +1,109 @@
+# Azure Functions
+## Overview
+Azure Functions is Microsoft’s cloud on-demand, serverless computing offering used for completing complex tasks via a collection of code written in a variety of programming or scripting languages. Azure Functions run code (also known as “functions”) that has been provided by the users, whilst Azure deploys and maintains the infrastructure to perform the tasks provided by the code. Every function has a trigger which can be time-based or manually run from an API call. Once triggered Azure will execute the code provided within the function.
+
+Microsoft are responsible for provisioning and maintaining the underlying infrastructure that Azure Functions utilises. This includes practices such as separating resources from other customers, ensuring regular updates of common software and runtimes that deployed functions run on and handling communication with other Azure services such as Azure Storage or Azure SQL that are done over Azure’s network. 
```

Comment:
Can we add any additional available context on how functions are isolated? Is there any information in the documentation that can provide some info on how functions are handled when they go cold or while they're kept warm.